Resources/Prototypes/Entities/Mobs/Customization/Markings/reptilian.yml

```
@@ -87,44 +87,54 @@
   markingCategory: HeadTop
   speciesRestriction: [Reptilian]
   sprites:
-  - sprite: Mobs/Customization/reptilian_parts.rsi
-    state: horns_curled
+  - sprite: _White/Mobs/Customization/reptilian_parts.rsi # WWDP-Edit
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)